src/MoBi.Presentation/Mappers/SelectedIndividualToIndividualSelectionDTOMapper.cs

```diff
@@ -0,0 +1,31 @@
+﻿using MoBi.Core.Domain.Model;
+using MoBi.Presentation.DTO;
+using OSPSuite.Core.Domain.Builder;
+using OSPSuite.Utility;
+
+namespace MoBi.Presentation.Mappers
+{
+   public interface ISelectedIndividualToIndividualSelectionDTOMapper : IMapper<IndividualBuildingBlock, IndividualSelectionDTO>
+   {
+
+   }
+
+   public class SelectedIndividualToIndividualSelectionDTOMapper : ISelectedIndividualToIndividualSelectionDTOMapper
+   {
+      private readonly IMoBiContext _moBiContext;
+
+      public SelectedIndividualToIndividualSelectionDTOMapper(IMoBiContext moBiContext)
+      {
+         _moBiContext = moBiContext;
+      }
+      
+      public IndividualSelectionDTO MapFrom(IndividualBuildingBlock individualBuildingBlock)
+      {
+         var individualSelectionDTO = new IndividualSelectionDTO(_moBiContext.CurrentProject.IndividualsCollection);
+         if(individualBuildingBlock != null)
+            individualSelectionDTO.SelectedIndividualBuildingBlock = individualBuildingBlock;
```

Comment:
this is weird. Why are you doing this if you have the Null individual Building block? the behavior of setting null could be in the setter of the DTO. (tell don't ask)